build/parser.mjs

```diff
@@ -959,18 +968,18 @@ class Parser {
       return;
     }
 
-    if (target.ReleaseDate) {
+    if (target?.ReleaseDate) {
       item.releaseDate = this.normalizeOggDate(target.ReleaseDate);
     }
-    if (target.VaultedDate) {
+    if (target?.VaultedDate) {
       item.vaultDate = this.normalizeOggDate(target.VaultedDate);
     }
-    if (target.EstimatedVaultedDate) {
+    if (target?.EstimatedVaultedDate) {
       item.estimatedVaultDate = this.normalizeOggDate(target.EstimatedVaultedDate);
     }
-    if (target.Vaulted) {
-      item.vaulted = target.Vaulted;
-    }
+
+    item.vaulted = target?.Vaulted ?? wikiaItem.vaulted;
+    if (item.vaulted === 'N/A') item.vaulted = true;
```

Comment:
Could you explain this line? Like in which case it would be N/A and why?

Reply:
i don't like much having this one line of code for one warframe but i guess there's no other way around it

Reply:
theoretically would also match other stuff if they ever move Vaulted to a field on item

Reply:
I would imagine for primes that come from baro since technically they aren't vaulted but also not unvaulted either